app/controllers/concerns/search.rb

```diff
@@ -6,11 +6,9 @@ module Search
 
   included do
     def scope_search
-      return policy_scope(resource) unless params[:search]
```

Comment:
Notice on the old version, `.paginate` was never called unless you passed the `search` param.